### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by jacobweinstock**
## Description

Adds Apache 2.0 License

## Why is this needed

Without it contributing is dodgy. 

Fixes: #58 

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [ ] added unit or e2e tests
- [ ] provided instructions on how to upgrade

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (micahhausler) [inline LICENSE:189]: nit: is this going to be filled out?

**Review by thebsdbox — CHANGES_REQUESTED** [review]
Can we fix the date on the footer?

**Comment by jacobweinstock** [comment]
> Can we fix the date on the footer? @jacobweinstock

do you mean something like add year but leave everything else? `Copyright 2021 [name of copyright owner]`?

File: LICENSE (hunk 1 above)
Comment (jacobweinstock) [reply LICENSE:189]: good question. does it need to be filled out? i don't understand this line very well. looking at other repos, some have it filled out and some don't.

not filled out:
https://github.com/kubernetes/kubernetes/blob/master/LICENSE

filled out:
https://github.com/kubernetes/minikube/blob/master/LICENSE

**Pull request merged**